feature-libs/order/root/facade/order.facade.ts

```diff
@@ -41,4 +44,13 @@ export abstract class OrderFacade {
    * Places an order
    */
   abstract placeOrder(termsChecked: boolean): Observable<Order>;
+  /**
+   * Return cart's pickup entries
+   */
+  abstract getPickupEntries(): Observable<OrderEntry[]>;
+
+  /**
+   * Return cart's delivery entries
+   */
+  abstract getDeliveryEntries(): Observable<OrderEntry[]>;
 }
```

Comment:
The doc comments mention the cart, should it be the order instead?

Reply:
yes